src/Rings.sol

```diff
@@ -0,0 +1,88 @@
+       b    .!        .= .?7???7~.     .>r .      .= 
+       .,.?4         , .^         1        `     4... 
+        J   ^         ,            5       `         ?<. 
+       .%.7;         .`     .,     .;                   .=. 
+       .+^ .,       .%      MML     F       .,             ?, 
+        P   ,,      J      .MMN     F        6               4. 
+        l    d,    ,       .MMM!   .t        ..               ,, 
+        ,    JMa..`         MMM`   .         .!                .; 
+         r   .M#            .M#   .%  .      .~                 ., 
+       dMMMNJ..!                 .P7!  .>    .         .         ,, 
+       .WMMMMMm  ?^..       ..,?! ..    ..   ,  Z7`        `?^..  ,, 
+          ?THB3       ?77?!        .Yr  .   .!   ?,              ?^C 
+            ?,                   .,^.` .%  .^      5. 
+              7,          .....?7     .^  ,`        ?. 
+                `<.                 .= .`'           1 
+                ....dn... ... ...,7..J=!7,           ., 
+             ..=     G.,7  ..,o..  .?    J.           F 
+           .J.  .^ ,,,t  ,^        ?^.  .^  `?~.      F 
+          r %J. $    5r J             ,r.1      .=.  .% 
+          r .77=?4.    ``,     l ., 1  .. <.       4., 
+          .$..    .X..   .n..  ., J. r .`  J.       `' 
+        .?`  .5        `` .%   .% .' L.'    t 
+        ,. ..1JL          .,   J .$.?`      . 
+                1.          .=` ` .J7??7<.. .; 
+                 JS..    ..^      L        7.: 
+                   `> ..       J.  4. 
+                    +   r `t   r ~=..G. 
+                    =   $  ,.  J 
+                    2   r   t  .; 
+              .,7!  r   t`7~..  j.. 
+              j   7~L...$=.?7r   r ;?1. 
+               8.      .=    j ..,^   .. 
+              r        G              . 
+            .,7,        j,           .>=. 
+         .J??,  `T....... %             .. 
+      ..^     <.  ~.    ,.             .D 
+    .?`        1   L     .7.........?Ti..l 
+   ,`           L  .    .%    .`!       `j, 
+ .^             .  ..   .`   .^  .?7!?7+. 1 
+.`              .  .`..`7.  .^  ,`      .i.; 
+.7<..........~<<3?7!`    4. r  `          G% 
+                          J.` .!           % 
+                            JiJ           .` 
+                              .1.         J 
+                                 ?1.     .'         
+                                     7<..%
+*/
+
+import "../lib/yield-utils-v2/contracts/mocks/ERC20Mock.sol";
+
+
+/// @title Rings, an ERC20 token
+/// @author hashedMae
+/// @notice This contract is for a new token called Rings
+/// @dev wip
+
+contract Rings is ERC20Mock {
+
+    constructor() ERC20Mock("Rings", "RNG") { }
+
+    /// @notice Mint some tokens
+    /// @dev Will mint tokens for whoever wants them
+    /// @param to The Address minted tokens will be sent to
+    /// @param amount The number of tokens to mint
+    function mint(address to, uint256 amount) public override {
```

Comment:
if mint and burn are in the ERC20Mock, then u dont need to add them here

i think the confusion comes from the fact that if u r making ur own ERC20Mock (and inheriting directly from ERC20) then u would need to write ur own mint/burn that calls the internal _mint / _burn